hack/install-tools.sh

```diff
@@ -52,16 +61,23 @@ if [ -z "$PY_CMD" ]; then
     fi
 fi
 
-ALT_BIN="$(realpath "${2:-.bin}")"
+ALT_BIN="${2:-.bin}"
```

Comment:
I just realizes that `.bin` is not listed in `.gitignore` at the root of this repo. Should add it for the case where `yamllint` is not installed on local host machine and installed locally within this repo `.bin` directory.

Reply:
done. added a commit to do this.